framework/Samplers/CustomSampler.py

```diff
@@ -66,6 +71,7 @@ def __init__(self):
     ForwardSampler.__init__(self)
     self.pointsToSample = {}
     self.infoFromCustom = {}
+    self.nameInSource = {} # dictionary to map the variable's sampled name to the name it has in Source
```

Comment:
we probably should create a general Alias system (Object dependent and not only Model dependent).
But for now it is fine

Reply:
I agree.  I think it might take a little effort to generalize for all the needs, but I believe it can be done.